rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java

```diff
@@ -190,6 +196,16 @@ public void startNodeImpl(final HostTracker hostTracker, final int nbNodes, fina
         final String nodeName = nodeNameBuilder.generateNodeName(hostTracker);
         clb.setNodeName(nodeName);
         clb.setNumberOfNodes(nbNodes);
+
+        // set the stratup script retrieved from NodeSource.properties
+        if (this.startupScript != null && !this.startupScript.isEmpty()) {
+            clb.setStartupScript(this.startupScript);
+        } else {
+            // in this case CommandlineBuilder will automatically fallback to creating the command as a list
+            // as a result, user's stratup script is disregarded and replaced by a default one
+            logger.error("Unable to correctly retrieve the startupScript for the node source " + nodeSource.getName());
```

Comment:
why is there a logger error here ? is the startup script never null if we did not enter it ?

Reply:
I still don't understand why there is a logger.error, I would suggest rather:
logger.info("Node source " + nodeSource.getName() + " is not using a startup script");